CONTRIBUTING.md

```diff
@@ -91,8 +91,9 @@ Here are some rules of thumb in no particular order:
 - Use `println()` only in CLI modules and `log` in library modules.
 
 We aim to have automated checks for these rules, but are bound to what [detekt](https://github.com/detekt/detekt) /
-[ktlint](https://github.com/pinterest/ktlint) support. In any case, please [run the `detekt` task](./README.md#development)
-locally before contributing.
+[ktlint](https://github.com/pinterest/ktlint) support. In any case, please
+[run the `detekt` task](https://github.com/oss-review-toolkit/ort/blob/main/README.md#development) locally before
```

Comment:
Should we maybe just drop the link to not confuse contributors to e.g. the ORT workbench?

Reply:
Yes. Done.